framework/OutStreams/Factory.py

```diff
@@ -22,29 +22,22 @@
 
 ################################################################################
 from utils import utils
-from .OutStreamManager import OutStreamManager
-from .OutStreamPlot import OutStreamPlot
-from .OutStreamPrint import OutStreamPrint
-## [ Add new class here ]
-################################################################################
-## Alternatively, to fully automate this file:
-# from OutStreamManagers import *
-################################################################################
 
-"""
-  Interface Dictionary (factory) (private)
-"""
-# This machinery will automatically populate the "knownTypes" given the
-# imports defined above.
-__base = 'OutStreamManager'
-__interFaceDict = {}
+from .OutStreamBase import OutStreamBase
+
+from .FilePrint import FilePrint
+from .GeneralPlot import GeneralPlot
+# from .DataMining import DataMining
+# from .VDCComparison import VDCComparison
```

Comment:
These are placeholders for the future; they can be removed if desired. Same with later on in this file.